docs/serving/configuration/config-defaults.md

```diff
@@ -30,319 +32,497 @@ data:
 the pod receives.
 {% endraw %}
 
-**Key**: `container-concurrency`
-
-**Default**: `"0"`
+* **Global key:** `container-concurrency`
+* **Per-revision annotation key:** `serving.knative.dev/container-concurrency`
+* **Possible values:** integer
+* **Default:** `"0"`
 
 **Example:**
-```yaml
-apiVersion:  v1
-kind:  ConfigMap
-metadata:
-  name:  config-defaults
-  namespace:  knative-serving
-data:
-  container-concurrency: "0"
-```
 
-### Container Concurrency Max Limit
+=== "Global (ConfigMap)"
+    ```yaml
+    apiVersion:  v1
+    kind:  ConfigMap
+    metadata:
+      name:  config-defaults
+      namespace:  knative-serving
+    data:
+      container-concurrency: "0"
+    ```
+
+=== "Per Revision"
+    ```yaml
+    apiVersion: serving.knative.dev/v1
+    kind: Service
+    metadata:
+      name: helloworld-go
+      namespace: default
+    spec:
+      template:
+        metadata:
+          annotations:
+            serving.knative.dev/container-name-template: "0"
+        spec:
+          containers:
+            - image: gcr.io/knative-samples/helloworld-go
+    ```
+
+### Container concurrency max limit
 
 {% raw %}
 
-The container concurrency max limit is an operator setting ensuring that
-the individual revisions cannot have arbitrary large concurrency
-values, or autoscaling targets. `container-concurrency` default setting
-must be at or below this value.
-
-Must be greater than 1.
+The `container-concurrency-max-limit` is an Operator setting that disables arbitrary large concurrency values, or autoscaling targets, for individual revisions. The `container-concurrency` default setting must be at or below this value. The value of the `container-concurrency-max-limit` setting must be greater than 1.
 
 !!! note
-    Even with this set, a user can choose a `containerConcurrency` value of 0 (unbounded) unless `allow-container-concurrency-zero` is set to "false".
+    Even with this set, a user can choose a `containerConcurrency` value of zero (unbounded), unless `allow-container-concurrency-zero` is set to `"false"`.
 
 {% endraw %}
 
-**Key**: `container-concurrency-max-limit`
-
-**Default**: `"1000"`
+* **Global key:** `container-concurrency-max-limit`
+* **Per-revision annotation key:** N/A
+* **Possible values:** integer
+* **Default:** `"1000"`
 
 **Example:**
-```yaml
-apiVersion:  v1
-kind:  ConfigMap
-metadata:
-  name:  config-defaults
-  namespace:  knative-serving
-data:
-  container-concurrency-max-limit: "1000"
-```
 
-### Allow Container Concurrency Zero
+=== "Global (Operator)"
+    ```yaml
+    apiVersion: operator.knative.dev/v1alpha1
+    kind: KnativeServing
+    metadata:
+      name: knative-serving
+    spec:
+      config:
+        data:
+          container-concurrency-max-limit: "1000"
```

Comment:
cc @houshengbo can you confirm how to configure this for the operator

Reply:
This is a key in the CM called config-defaults, so it should be 
```
    apiVersion: operator.knative.dev/v1beta1
    kind: KnativeServing
    metadata:
      name: knative-serving
      namespace: knative-serving
    spec:
      config:
        defaults:
          container-concurrency-max-limit: "1000"
```

In addition, add the field `namespace` and change the CR version into v1beta1.